test/unit/math/test_ad.hpp

```diff
@@ -1565,6 +1642,111 @@ void expect_common_binary(const F& f, bool disable_lhs_int = false) {
     }
 }
 
+std::vector<double> common_complex_parts_re() {
+  return {-4, -2.5, -1.5, -0.3, -0.1, 0.1, 1.3, 2.1, 3.9};
+}
+
+std::vector<double> common_complex_parts_im() {
+  return {-4, -2.5, -1.5, -0.3, -0.0, 0.0, 1.3, 2.1, 3.9};
+}
+
+std::vector<std::complex<double>> common_complex() {
+  std::vector<std::complex<double>> zs;
+  auto complex_re = common_complex_parts_re();
+  auto complex_im = common_complex_parts_im();
+  for (int i = 0; i < complex_re.size(); ++i) {
+    zs.emplace_back(complex_re[i], complex_im[i]);
+  }
+  /*
+  for (double re : common_complex_parts_re())
```

Comment:
Just remove commented-out code.  (Bonus:  commenting out code should always be done with line-based comments so you can easily scan the scope of comments.)